.eslintignore

```diff
@@ -1,5 +1,12 @@
-.circleci/*
-.githooks/*
-.idea/*
-.vscode/*
-node_modules/*
+.circleci
+.githooks
+.idea
+.vscode
+.yarn*
+dist
+node_modules
+*.d.ts
```

Comment:
Why wouldn't we want to lint these?

Reply:
Agreed, though the original lint command in the Makefile excluded *.d.ts files in the various grep filters, hence the addition here.  I'll remove line 8 in this case.